### DISCUSSION TITLE
Few eza icons different compared to exa icons

### DISCUSSION THREAD

**Discussion by M0hanrajp** [eza-community/eza#999] [Q&A, Howto, Help] [answered]
**[ Issue ]**

Few eza icons seem to look invalid compared to exa icons.

Please have a look at the below screenshot: **Left: exa & right: eza**

![image](https://github.com/eza-community/eza/assets/168116222/12c6a85e-4e81-4666-86df-bcf76ae93996)

Here have look at the file named `.bash_history`, `.bash_logout`, `.bashrc` & `.profile` on the right which are output\
from eza using the command `alias ls='eza -lagh --time-style=long-iso --group-directories-first --icons'` they\
are all displayed with icon `#!` which seem invalid as compared to left output using exa.

Is this expected outcome using eza ?

---

Thanks,
Please let me know if further information is required.

**Comment by gierens** [comment, marked as answer]
Yes, this is intentional.

We refactored around a lot in the icons module and in the process also added and changed icons and filename/extension bindings based on user requests and also our own judgement. Some of the conversation around this can be found here #196, and the changes by searching for "icons" in the PRs (https://github.com/eza-community/eza/pulls?q=is%3Apr+icons+) or the `CHANGELOG.md` (https://github.com/eza-community/eza/blob/main/CHANGELOG.md).

Whether we improved things is obviously partly subjective and a matter of taste, but that is one reason we are currently working on a config file feature that would also allow users to override icons, see #840

**Reply by M0hanrajp** [reply]
Ok Thanks for your response.
